modules/rosa-sts-cluster-terraform-destroy.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly-terraform.adoc
+//
+:_content-type: PROCEDURE
+
+[id="sd-terraform-cluster-destroy_{context}"]
+= Deleting your ROSA cluster with Terraform
+
+Terraform uses the `destroy` command to remove all resources that you created with the `terraform apply` command.
+
+[NOTE]
+====
+You should not modify your Terraform .tf files nor the "terraform.tfvars" file before destroying your resources. Terraform uses these variables to select the correct resources to delete.
+====
+
+.Procedure
+. In the directory where you ran `terraform apply` to create your cluster, run the following command to delete the cluster:
```

Comment:
Do we use numbered steps in procedures when we only have one step? I feel that I was informed to use a bullet in this case, but I could be wrong....seen both (I think).